### ISSUE TITLE
Class output doesn't seem to work with RGB function

### ISSUE THREAD

**Issue by sebpowell** [brattonross/windy-radix-palette#25]
Hello,

Thank you for this library. I've been trying to get it working using the following setup: CRA v5 and Tailwind 3.3.3. 

First of all, it would appear the instructions detailed in the README are out of date – the only way I was able to get it to work without throwing an error is as such:

```TS
const { withTV } = require("tailwind-variants/transformer");

const radix = require("windy-radix-palette");

module.exports = withTV({
  content: ["./src/**/*.{js,jsx,ts,tsx}", "./stories/**/*.{js,ts,jsx,tsx}"],
  plugins: [
    radix({opacitySupport: true}),
  ],
});
```
Either that, or my setup doesn't work with the recommended approach for whatever reason.

This seems to work, in that a bunch of CSS variables are created and added to the `:root selector`, see here:

<img width="945" alt="Screenshot 2023-10-11 at 19 19 59" src="https://github.com/brattonross/windy-radix-palette/assets/1786366/e3f8a8d4-50fd-453a-a15e-74d9a4060a5e">

From what I gather, the alpha for color functions is defined in the following way (see https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color):

```
color(display-p3 1 0.5 0 / .5);
```

The classes that are generated by this package look like this:

<img width="709" alt="Screenshot 2023-10-11 at 19 22 41" src="https://github.com/brattonross/windy-radix-palette/assets/1786366/dfe9661f-bd48-4455-992b-e801e804b6b7">

Applying this to any HTML element on the page doesn't actually turn the background red. 

Changing the `background-color` attribute to simply `var(--red-9)` works fine. So I'm guessing the issue is that variables that use the color function cannot be passed in to the `rgb` function. I did a bit of Googling but couldn't find any clear answers – wondering whether I'm doing something wrong, or whether this is in fact a bug?

Thank you in advance.

**Comment by brattonross** [comment]
Hey, thanks for reporting your issue -- do you mind sharing which version of the package you are using?

**Comment by sebpowell** [comment]
@brattonross thanks for coming back to me. The above was using the latest version, which I believe was [v2.0.0-beta.0](https://github.com/brattonross/windy-radix-palette/releases/tag/v2.0.0-beta.0). Let me know if you need any other info.

**Comment by brattonross** [comment]
Hi, I've tried to recreate your issue, but I'm not seeing the problems with getting the plugin setup that you are seeing. Take a look at [this codesandbox](https://codesandbox.io/p/github/brattonross/wrp-issue-cra/main) where I've tried to make a minimal repro. I got the plugin working with the instructions in the readme, so I'm not sure what is different about your setup that might cause an error to happen.

I suspect that if we manage to get you using the plugin in the expected way, then your issue with the color opacity might get fixed as a side effect. To me it looks like the part of the plugin that generates the CSS variables thinks that opacity support is disabled, but the part that generates the Tailwind config thinks that it is enabled.
